commons_boundaries/boundary_config.py

```diff
@@ -0,0 +1,9 @@
+# Module for boundaryConfig class.
+
+from pathlib import Path
+
+
+class BoundaryConfig(object):
+    """docstring for BoundaryConfig"""
```

Comment:
I think if the docstring isn't going to be more descriptive than the class name, it's probably best to just leave it out ;)